playwright-e2e/pages/brain/base-page.ts

```diff
@@ -0,0 +1,13 @@
+import { Page } from '@playwright/test';
+import PageBase from 'pages/page-base';
+
+const { BRAIN_BASE_URL } = process.env;
+
+export abstract class BrainBasePage extends PageBase {
+  protected constructor(page: Page) {
+    if (BRAIN_BASE_URL == null) {
+      throw Error(`Invalid Brain base URL: process.env.BRAIN_BASE_URL=${BRAIN_BASE_URL}`);
```

Comment:
Nice - I remember when setting up on different IDEs I kept forgetting to fill out that info in the env file and wondered what was wrong